libnavigation-core/src/main/java/com/mapbox/navigation/core/routerefresh/RouteRefreshController.kt

```diff
@@ -62,7 +62,7 @@ class RouteRefreshController internal constructor(
         }
         plannedRouteRefreshController.pause()
         immediateRouteRefreshController.requestRoutesRefresh(routes) {
-            if (it.value?.anySuccess() == false) {
+            if (it.isValue) {
```

Comment:
why don't we resume in case of failture?

Reply:
Null here doesn't mean failure. Null means it wasn't executed because a newer request came along.

Reply:
does null mean that another request was scheduled and current is cancelled so that we don't need to resume planned refreshes?

Reply:
Yeah, because we'll think of that when the new request finishes, this on is just replaced by a new one. 
It's not super clear, I think it'd be better to introduce some descriptive enum instead of Expected.

Reply:
> I think it'd be better to introduce some descriptive enum instead of Expected.

Great idea 👍